src/interfaces/alteration-interfaces/alteration-interface.ts

```diff
@@ -1,20 +1,24 @@
-import { BusinessInformationIF, CourtOrderIF, NameRequestIF, NameTranslationIF, ShareStructureIF }
+import { CourtOrderIF, NameRequestIF, NameTranslationIF, ShareStructureIF }
   from '@/interfaces/'
 import { CoopTypes } from '@/enums'
 import { ContactPointIF } from '@bcrs-shared-components/interfaces/'
+import { CorpTypeCd } from '@bcrs-shared-components/enums'
 
 //
 // Ref: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/alteration.json
 //
 export interface AlterationIF {
   provisionsRemoved?: boolean
-  business?: BusinessInformationIF
+  business: {
+    identifier: string
+    legalType: CorpTypeCd
+  }
```

Comment:
These are the only 2 business properties that we set.

Ditto for the other interfaces below.

(The alteration schema says this is a full business object, but obviously the back end schema check didn't validate it.)